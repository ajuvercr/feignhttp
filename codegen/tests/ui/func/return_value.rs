use feignhttp_codegen::get;

// error: function must have a return value
//    |  pub async fn send_get() {}
//    |      ^^^^^^^^^^^^^^^^^^^

#[get("http://xxx")]
pub async fn send_get() {}

fn main() {}
