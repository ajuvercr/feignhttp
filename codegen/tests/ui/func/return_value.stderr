error: function must have a return value
 --> tests/ui/func/return_value.rs:8:5
  |
8 | pub async fn send_get() {}
  |     ^^^^^^^^^^^^^^^^^^^
